io_scene_godot/converters/mesh.py

```diff
@@ -105,85 +67,6 @@ def triangulate_mesh(mesh):
     mesh.update(calc_tessface=True)
 
 
-def find_bone_vertex_groups(vertex_groups, skeleton_node):
-    """Find the id of vertex groups connected to bone weights,
-    return a dict() mapping from vertex_group id to bone id"""
-    ret = dict()
-    if skeleton_node is not None:
-        # the bone's index in the bones list is exported as the id
-        for bone_name, bone_id in skeleton_node.bone_name_to_id_map.items():
-            group = vertex_groups.get(bone_name)
-            if group is not None:
-                ret[group.index] = bone_id
-    return ret
-
-
-def make_arrays(escn_file, export_settings, skeleton_node, node):
-    """Generates arrays of positions, normals etc"""
-    armature_data = armature.get_armature_data(node)
-    if armature_data is not None:
-        original_pose_position = armature_data.pose_position
-        armature_data.pose_position = 'REST'
-        bpy.context.scene.update()
-
-    if not export_settings['use_mesh_modifiers']:
-        for modifier in node.modifiers:
-            if not isinstance(modifier, bpy.types.ArmatureModifier):
-                modifier.show_render = False
-
-    mesh = node.to_mesh(bpy.context.scene,
-                        True,
-                        "RENDER")
-
-    # Prepare the mesh for export
-    triangulate_mesh(mesh)
-
-    uv_layer_count = len(mesh.uv_textures)
-    if uv_layer_count > 2:
-        uv_layer_count = 2
-
-    if mesh.uv_textures:
-        has_tangents = True
-        mesh.calc_tangents()
-    else:
-        mesh.calc_normals_split()
-        has_tangents = False
-
-    # find the vertex group id of bone weights
-    gid_to_bid_map = find_bone_vertex_groups(
-        node.vertex_groups, skeleton_node)
-
-    # Separate by materials into single-material surfaces
-    surfaces = generate_surfaces(
-        escn_file,
-        export_settings,
-        mesh,
-        has_tangents,
-        gid_to_bid_map
-    )
-
-    if (export_settings['export_shape_key'] and
-            node.data.shape_keys is not None):
-        export_morphs(
-            export_settings, node, surfaces, has_tangents, gid_to_bid_map)
-
-    has_bone = True if armature_data is not None else False
-
-    for surface_id, surface in enumerate(surfaces):
-        surface.id = surface_id
-        surface.vertex_data.has_bone = has_bone
-        for vert_data in surface.morph_arrays:
-            vert_data.has_bone = has_bone
-
-    bpy.data.meshes.remove(mesh)
-
-    if armature_data is not None:
-        armature_data.pose_position = original_pose_position
-        bpy.context.scene.update()
-
-    return surfaces
-
-
 def extract_shape_keys(blender_shape_keys):
```

Comment:
I think that this and `initialize_surfaces_morph_data` should be `@staticmethod`'s of the `MeshResourceExporter` class